### PR TITLE
507: Increase Test Coverage for CLI tool, Part 1

### PR DESCRIPTION
**Pull request by netme**
Related to #507

### CODE REVIEW

**Comment by netme** [comment]
Awesome, `codecov` already reports +4% :-)

File: cli/zally/utils/http_client_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		tests.AssertEquals(t, nil, err)
+
+		responseBody, _ := ioutil.ReadAll(response.Body)
+		tests.AssertEquals(t, "200 OK", response.Status)
+		tests.AssertEquals(t, "Hello", string(responseBody))
+	})
+
+	t.Run("fails_when_timeout_is_reached", func(t *testing.T) {
+		handler := func(w http.ResponseWriter, r *http.Request) {
+			time.Sleep(6 * time.Second)
```
Comment (maxim-tschumak) [inline cli/zally/utils/http_client_test.go]: Does it make sense to add a function `http_client:SetTimeout(t)` to make it configurable? (and adding the possibility to make this test faster)

**Comment by maxim-tschumak** [comment]
:+1:

File: cli/zally/utils/http_client_test.go (hunk 1 above)
Comment (netme) [reply cli/zally/utils/http_client_test.go]: Yeah, maybe we can move it to CLI properties, would be useful. I will create another PR later

**Comment by netme** [comment]
👍

**Pull request merged**